app/steps/ui/iht/value/index.js

```diff
@@ -4,14 +4,15 @@ const ValidationStep = require('app/core/steps/ValidationStep');
 const validator = require('validator');
 const numeral = require('numeral');
 const FieldError = require('app/components/error');
+const FeatureToggle = require('app/utils/FeatureToggle');
 
 class IhtValue extends ValidationStep {
 
     static getUrl() {
         return '/iht-value';
     }
 
-    handlePost(ctx, errors) {
+    handlePost(ctx, errors, formdata, session, hostname, featureToggles) {
```

Comment:
Can we have a unit test for at least the new variable that was added to this method